src/app/services/app-auth.service.ts

```diff
@@ -14,7 +15,7 @@ import { BehaviorSubject } from 'rxjs';
 })
 export class AppAuthService {
   private isAuthenticated$$ = new BehaviorSubject<boolean>(false);
```

Comment:
I think it could be removed now

Reply:
it is still used by isDoneLoading$$

Reply:
Where? I see only one place here 
![image](https://user-images.githubusercontent.com/1961590/133980936-097f5405-4bb0-4bf8-9398-404c5e33c12f.png)